### PR TITLE
Add Encryption and Decryption sample

### PR DESCRIPTION
**Pull request by SharonHart**
## Change Description

Add a sample for anonymized decryption and encryption of text

## Checklist

- [x] I have reviewed the [contribution guidelines](https://github.com/microsoft/presidio/blob/main/CONTRIBUTING.md)
- [x] I have signed the CLA
- [x] My code includes unit tests
- [x] All unit tests and lint checks pass locally
- [x] My PR contains documentation updates / additions if required

### CODE REVIEW

File: docs/samples/python/encrypt_decrypt.ipynb [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+   "source": [
+    "# Initialize the engine:\n",
+    "engine = DeanonymizeEngine()\n",
+    "\n",
+    "# Invoke the deanonymize function with the text, anonymizer results\n",
+    "# and a 'decrypt' operator to get the original text as output.\n",
+    "result = engine.deanonymize(\n",
+    "    text=anonymized_text,\n",
+    "    entities=[\n",
+    "        AnonymizerResult(start=11, end=55, entity_type=\"PERSON\"),\n",
```
Comment (omri374) [inline docs/samples/python/encrypt_decrypt.ipynb:155]: Consider showing how to extract the anonymized text from the anonymizer output (not fetch only the text, also use the indices)

**Comment by omri374** [comment]
Looks great! One minor comment.

File: docs/samples/python/encrypt_decrypt.ipynb (hunk 1 above)
Comment (SharonHart) [reply docs/samples/python/encrypt_decrypt.ipynb:155]: And calling an internal method to decrypt?

Comment (SharonHart) [reply docs/samples/python/encrypt_decrypt.ipynb:167]: @omri374 It was, done, good call

**Review by omri374 — APPROVED** [review]

File: docs/samples/python/encrypt_decrypt.ipynb (hunk 1 above)
Comment (omri374) [reply docs/samples/python/encrypt_decrypt.ipynb:167]: Awesome

**Review by navalev — APPROVED** [review]

**Pull request merged**